org.civicrm.reqangular/src/common/decorators/angular-date/xeditable-addon.js

```diff
@@ -0,0 +1,26 @@
+/* eslint-env amd */
+
+define([], function () {
+  'use strict';
+
+  /**
+   * Decorates the xeditable directive
```

Comment:
too generic, say how you are decorating it

Reply:
Done